MekHQ/src/mekhq/campaign/parts/MissingBayDoor.java

```diff
@@ -72,10 +72,15 @@ public void fix() {
             unit.addPart(actualReplacement);
             campaign.getQuartermaster().addPart(actualReplacement, 0);
             replacement.decrementQuantity();
+
+            // Calling 'remove()' has the side effect of setting this.parentPart to null.
+            // Issue #2878
+            Part _parentPart = parentPart;
```

Comment:
Sure - not a problem.  Actually, "Part" is implied by type, it's redundant.  Do you mind if I call it `parentReference`?  Or `parentRef`?

Also, do you want me to remove the comments?  I would still want to put a simple `// sets this.parentPart to null.` after the `remove()` call because anyone reading this code should know that and it's not obvious.

Reply:
Let's leave the comments. parentReference is fine.